### PR TITLE
feat(MessageView): hide image url in the message when image is unfurled

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

The url is hidden when there is only one url in the message with no additional text and the image is shown (unfurled).

Closes: #7321
Depends on: https://github.com/status-im/StatusQ/pull/916

### Affected areas

MessageView, LinksMessageView, StatusMessage

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Kazam_screencast_00003.webm](https://user-images.githubusercontent.com/20650004/191289692-3982e249-206e-4c2a-873f-4148b972486f.webm)

[Kazam_screencast_00005.webm](https://user-images.githubusercontent.com/20650004/191290682-3f32a996-9a43-41a8-89af-8c4873d67a9d.webm)

### CODE REVIEW

File: ui/imports/shared/views/chat/MessageView.qml [hunk 1]
```diff
@@ -234,6 +234,11 @@ Loader {
 
         readonly property int chatButtonSize: 32
 
+        property bool isSingleImage: linkUrlsModel.count === 1 && linkUrlsModel.get(0).isImage
```
Comment (caybro) [inline ui/imports/shared/views/chat/MessageView.qml:237]: `readonly`

**Review by caybro — APPROVED** [review]
Nice work!

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Review by faust4exe — APPROVED** [review]
LGTM

**Pull request merged**